### PR TITLE
[Tech Debt] Remove Apple Watch code

### PR DESCRIPTION
**Pull request by kcw-grunt**
### Summary 
Originally this was a small PR for the Apple Watch code. But, there were some other items that needed to be addressed.  This has been rebased to `release/v2.8.0`
 
#### Changes:
 - Support Email
 - Dead Code NotificationServiceExtension
 - HTML for Support Email 
- The Apple Watch Code is very old and has never been updated.
- There is a greater plan to remove superfluous code.
- If others ask to re-add it we will look again with the later versions
 
| Support Email |
|---------------|
|supportt|

### CODE REVIEW

File: loafwallet/src/ApplicationController.swift [hunk 1]
```diff
@@ -216,8 +215,7 @@ class ApplicationController : Subscriber, Trackable {
                     }
                 }
                 exchangeUpdater?.refresh(completion: {
```
Comment (kcw-grunt) [inline loafwallet/src/ApplicationController.swift:217]: Will eventually remove this call. : `exchangeUpdater?.refresh(completion: {})`

Currently there is a chain dependency on the labels so it has been kept in for the time being

File: loafwallet/src/ApplicationController.swift [hunk 2]
```diff
@@ -268,8 +266,7 @@ class ApplicationController : Subscriber, Trackable {
             defaultsUpdater?.refresh()
             walletManager?.apiClient?.events?.up()
             exchangeUpdater?.refresh(completion: {
```
Comment (kcw-grunt) [inline loafwallet/src/ApplicationController.swift:268]: Will eventually remove this call. : `exchangeUpdater?.refresh(completion: {})`

Currently there is a chain dependency on the labels so it has been kept in for the time being

**Comment by kcw-grunt** [comment]
@mosadialiou 
Build is awaiting in App Store 
Screen Shot 2020-10-17 at 1 51 00 AM

File: loafwallet/src/Constants/Constants.swift [hunk 3]
```diff
@@ -80,11 +80,71 @@ struct C {
...
+                <td colspan="2" align="left" style="padding-top:7px; padding-bottom:7px; border-top: 3px solid #777; border-bottom: 1px dotted #777;">
+                    <span style="font-size: 13; line-height: 16px;" face="'Lucida Grande',Verdana,Arial,sans-serif">
+                        <div>Please reply to this email with the following information so that we can prepare to help you solve your Litewallet issue.</div>
+                      <br>
+                         <div>
+                            1. What is the platform of your mobile device (Android or iOS)?</div>
+                          <br>
+                          <br>
+                            <div>
+                            2. What version of software running on your mobile device (e.g.; Android 20, or 24 or iOS 13.7 or iOS 14)?</div>
```
Comment (mosadialiou) [inline loafwallet/src/Constants/Constants.swift:129]: There is no Android 20 or 24 instead it is Android 9, 10 and 11 (released september 8, 2020)

**Review by mosadialiou — CHANGES_REQUESTED** [review]

File: loafwallet/src/Constants/Constants.swift (hunk 3 above)
Comment (kcw-grunt) [reply loafwallet/src/Constants/Constants.swift:129]: Got it thanks. (These are meant to be examples and in fact I will give you a version for Android without iOS)

**Review by mosadialiou — APPROVED** [review]
LGTM

**Pull request merged**